spec/controllers/state_file/archived_intake/archived_intake_controller_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+describe StateFile::ArchivedIntakes::ArchivedIntakeController, type: :controller do
+  let(:ip_address) { '192.168.0.1' }
+  let(:email_address) { 'test@example.com' }
+  let(:request_instance) { instance_double(StateFileArchivedIntakeRequest) }
```

Comment:
unless you're mocking methods on this model instance (which i don't see anywhere), i think it would make more sense to just create an actual `StateFileArchivedIntakeRequest` from the factory
```suggestion
  let(:request_instance) { create :state_file_archived_intake_request, ip_address: ip_address }
```